homeworks/week02/README.md

```diff
@@ -0,0 +1,232 @@
+# Week 2, Homework
+
+For this homework you will write code that takes some data and renders
+it into a web page, using functions, objects and arrays. The data
+will be a list of movies with some information about each. You will
+receive this in the form of an array of simple JavaScript objects,
+which your code will convert into HTML. You will also add some simple
+CSS to style the page.
```

Comment:
Might be worth explicitly pointing out that the Javascript _code_ will _write_ HTML _code_ as a _string_. Metaprogramming might be a conceptual leap for some.